resources/views/event/display_includes/event_block.blade.php

```diff
@@ -83,6 +83,19 @@
                 </span>
             @endif
 
+            @if($event->activity && $event->activity->users->count()>0)
+               <br>
+               <span>
+                   @if($event->activity->isFull())
+                       <i class="fas fa-user-alt fa-fw" style="color: orange;" aria-hidden="true"></i>
+                       Full!
```

Comment:
Maybe add something to incentivize members to sign up for the backup list if it is available? Otherwise people might see that an event is full and not bother looking at the event anymore.

Reply:
I agree that that would be a nice addition but do not have a clear solution for it. Do you have any input?